portal/src/texts/profile/animation.mdx

```diff
@@ -37,22 +41,22 @@ Ved hjelp av easingkurver og målrettet bruk av parallakseffekter, skaper vi smi
 />
 
 ### Posisjon
-Ved interaksjon, flyttes komponenter til en definert posisjon eller fikseres i en posisjon. 
+Hvis elementer flyttes, må dette skje på en forventet måte som har sammenheng med brukerens handling.
 Dette hjelper brukeren til å lettere navigere på siden. 
```

Comment:
Dette gjør det lettere for brukeren å navigere på siden.

Reply:
Bra innspill. Korrigert nå.